evb

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -e
+
+get_path () {
+  # calculates the 'root' path to run the container
+  if [ "${1}" == '' ] ; then
+    echo "ERROR: ensure you start this script either in a openvox control or a module directory" >&2
+    exit 200
+  elif [ -f "${1}/metadata.json" ] || [ -f "${1}/Puppetfile" ] ; then 
+    echo $1
+  else
+    get_path $(echo $1|sed 's|/[^/]*$||')
+  fi
+}
+
+usage() {
+  # print usage of this script and exit
+  cat << EOF
+Usage: ${0} [options] [command]
+
+available options:
+  --noop        : print the command to run, but do not run it
+  --entrypoint  : use a different entrypoint
+                  examples for available endpoints are: 
+                  onceover, 
+                  default: no entrypoint sepcified
+  --image image : use a different image (default ${PODIMAGE})
+  --env VAR=val : specify environment variables (can be used multiple times)
+                  Remark: the term './' in a assignment will be replaced with
+                  the correct path to be used in the container.
+                  Example: if you start the script in ~/openvox-supermodule/spec/classes
+                           and set --env SPEC=./supermodule_spec.rb we will
+                           run VoxBox with -e SPEC=spec/classes/supermodule_spec.rb
+  --volume vol  : specify an additional volume to put into the container
+                  see podman man page how to specify 'vol'. (no path magic is done ;)) 
+
+available command:
+   help  : print this help message and exit
+   
+Commands/Options no listed here are passed to voxbox as is.
+use the '--noop' option to print the detailed call to VoxBox.
+EOF
+  exit
+}
+
+run_container () {
+  P=$(get_path $PWD)
+  command="podman run -it --rm $(sed "s|=${P}/|=|g" <<< "${PODENV}") -v ${P}:/repo:Z ${PODOPTIONS} ${PODIMAGE} ${VOXBOXCOMMAND}"
+  if [ ${NOOP} -eq 0 ] ; then
+     ${command}
+  else
+     echo command to run:
+     echo ${command}
+     echo
+  fi
+}
+
+# get options
+NOOP=0
+PODOPTIONS=''
+PODENV=''
+PODIMAGE='ghcr.io/voxpupuli/voxbox:8'
+VOXBOXCOMMAND=''
+while [[ $# -gt 0 ]] do
```

Comment:
most likely should be `while [[ $# -gt 0 ]]; do` else i just got an syntax error on MacOS.